config_rb_reader_test.go

```diff
@@ -0,0 +1,35 @@
+package chef
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestNewClientRb(t *testing.T) {
+	clientRegistry["client_key"] = func(s []string, path string, m *ConfigRb) {
+		str := StringParserForMeta(s)
+		data := strings.Split(str, "/")
+		m.ClientKey = data[len(data)-1]
+	}
+	data := `current_dir = File.dirname(__FILE__)
+		log_level                :info
+		log_location             STDOUT
+		node_name                "test"
+		client_key               "#{current_dir}/test.pem"
+		chef_server_url          "https://server/organizations/test"
+		cookbook_path            ["#{current_dir}/../cookbooks"]`
+	cb, err := NewClientRb(data, "")
+	if err != nil {
```

Comment:
Let's use the `assert` package for these checks, so that we're consistent with the rest of the library.  Same in other test files. You can find examples in the existing tests.

Reply:
Done